tools/create_release_files.sh

```diff
@@ -36,57 +58,59 @@ if [[ `basename $CURR_DIR` != "tools" ]] ; then
   exit 1
 fi
 
+##
+## Variables with defaults (if not overwritten by environment)
+##
 GPG_PASSPHRASE=${GPG_PASSPHRASE:-XXX}
 GPG_KEY=${GPG_KEY:-XXX}
+GIT_AUTHOR=${GIT_AUTHOR:-"Your name <you@apache.org>"}
 GIT_BRANCH=${GIT_BRANCH:-branch-1.0}
 OLD_VERSION=${OLD_VERSION:-0.6-incubating-SNAPSHOT}
 RELEASE_VERSION=${NEW_VERSION}
 RELEASE_CANDIDATE=${RELEASE_CANDIDATE:-rc1}
 NEW_VERSION_HADOOP1=${NEW_VERSION_HADOOP1:-"$RELEASE_VERSION-hadoop1"}
-USER_NAME=${USER_NAME:-pwendell}
+USER_NAME=${USER_NAME:-yourapacheidhere}
 MVN=${MVN:-mvn}
 GPG=${GPG:-gpg}
-SHASUM=${SHASUM:-sha512sum}
-MD5SUM=${MD5SUM:-md5sum}
-sonatype_user=${sonatype_user:-rmetzger} #legacy variable name referring to maven
+sonatype_user=${sonatype_user:-yourapacheidhere}
 sonatype_pw=${sonatype_pw:-XXX}
 
-#echo $NEW_VERSION_HADOOP1
-#sleep 5
-#set -e
 
-# create source package
+if [ "$(uname)" == "Darwin" ]; then
+    SHASUM=shasum
+    MD5SUM=md5
```

Comment:
Well, the important thing is to have the checksum to be able to verify with
any kind of md5 calculation tool. But I changed it for consistency. Thanks!

On Wed, Jun 10, 2015 at 10:51 AM, Ufuk Celebi <notifications@github.com>
wrote:

> In tools/create_release_files.sh
> <https://github.com/apache/flink/pull/811#discussion_r32098664>:
>
> >
> > -# create source package
> > +if [ "$(uname)" == "Darwin" ]; then
> > +    SHASUM=shasum
> > +    MD5SUM=md5
>
> I didn't know that md5sum correctly works with the reversed files as well.
> So this is optional.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/flink/pull/811/files#r32098664>.
>